satpy/etc/composites/viirs.yaml

```diff
@@ -231,10 +231,10 @@ composites:
     - name: M05
       modifiers: [sunz_corrected, rayleigh_corrected]
     optional_prerequisites:
-    - name: I01
+    - name: I02
       modifiers: [sunz_corrected_iband, rayleigh_corrected_iband]
     standard_name: false_color
-    high_resolution_band: blue
+    high_resolution_band: green
```

Comment:
Out of curiosity, what was the reason for changing bands here? I presume to match some external visualisation method?

Reply:
It matches what we do in Polar2Grid. The findings are basically that the image looks better when you do it on the green band. I don't remember the scientific reasoning, but can I use the excuse of "Liam Gumley told me to do it"?

Reply:
Yeah, let's go with that ;-)